src/cli.c

```diff
@@ -165,6 +167,14 @@ int main(int argc, char *argv[])
                     print_help();
                     return 1;
                 }
+            } else if (SDL_strcmp(arg, "-i") == 0 || SDL_strcmp(arg, "--include") == 0) {
+                if (i + 1 >= argc) {
+                    SDL_LogError(SDL_LOG_CATEGORY_APPLICATION, "%s requires an argument", arg);
+                    print_help();
+                    return 1;
+                }
+                i += 1;
+                includeDir = argv[i];
```

Comment:
We don't warn about this right now, but would it make sense to warn/error when `-i` is passed more then once?
e.g.
```suggestion
            } else if (SDL_strcmp(arg, "-i") == 0 || SDL_strcmp(arg, "--include") == 0) {
                if (includeDir) {
                    SDL_LogError(SDL_LOG_CATEGORY_APPLICATION, "'%s' can only be used once", arg);
                    print_help();
                    return 1;
                }
                if (i + 1 >= argc) {
                    SDL_LogError(SDL_LOG_CATEGORY_APPLICATION, "%s requires an argument", arg);
                    print_help();
                    return 1;
                }
                i += 1;
                includeDir = argv[i];
```